dist/constants.json

```diff
@@ -5883,931 +5883,931 @@
         "value": "437GJ",
-        "label": "DICKINSON VA CLINIC"
+        "label": "Dickinson VA Clinic"
       },
       {
         "value": "437GK",
-        "label": "JAMESTOWN VA CLINIC"
+        "label": "Jamestown VA Clinic"
       },
       {
         "value": "437GL",
-        "label": "DEVILS LAKE VA CBOC"
+        "label": "Devils Lake VA Clinic"
       },
       {
         "value": "437",
-        "label": "FARGO VA HCS"
+        "label": "Fargo Regional Benefit Office"
       }
     ],
     "MN": [
       {
         "value": "437GC",
-        "label": "FERGUS FALLS"
+        "label": "Fergus Falls VA Clinic"
       },
       {
         "value": "618GA",
-        "label": "SOUTH CENTRAL CBOC"
+        "label": "St. James VA Clinic"
       },
       {
         "value": "618GB",
-        "label": "HIBBING (CBOC)"
+        "label": "Hibbing VA Clinic"
       },
       {
         "value": "618GD",
-        "label": "ST. PAUL (CBOC)"
+        "label": "Maplewood VA Clinic"
       },
       {
         "value": "618GG",
-        "label": "ROCHESTER (CBOC)"
+        "label": "Rochester VA Clinic"
       },
       {
         "value": "656GA",
-        "label": "BRAINERD VA COMMUNITY OUTPATIENT CLINIC"
+        "label": "Brainerd VA Clinic"
       },
       {
         "value": "656GB",
-        "label": "MONTEVIDEO CBOC"
+        "label": "Montevideo VA Clinic"
       },
       {
         "value": "437GE",
-        "label": "BEMIDJI CBOC"
+        "label": "Bemidji VA Clinic"
       },
       {
         "value": "656GC",
-        "label": "MAX J. BEILKE CBOC"
+        "label": "Max J. Beilke Department of Veterans Affairs Outpatient Clinic"
       },
       {
         "value": "618GI",
-        "label": "NORTHWEST METRO VA OPC"
+        "label": "Northwest Metro VA Clinic"
       },
       {
         "value": "618GK",
-        "label": "ALBERT LEA CBOC"
+        "label": "Albert Lea VA Clinic"
       },
       {
         "value": "618GL",
-        "label": "MINNEAPOLIS VA CBOC"
+        "label": "Minneapolis VA Clinic"
       },
       {
         "value": "618GJ",
-        "label": "SHAKOPEE CBOC"
+        "label": "Shakopee VA Clinic"
       },
       {
         "value": "618GN",
-        "label": "LYLE C PEARSON CBOC"
+        "label": "Lyle C. Pearson Community Based Outpatient Clinic"
       },
       {
         "value": "618QB",
-        "label": "ELY VA CLINIC"
+        "label": "Ely VA Clinic"
       },
       {
         "value": "618",
-        "label": "MINNEAPOLIS VA HCS"
+        "label": "Minneapolis VA Medical Center"
       },
       {
         "value": "656",
-        "label": "ST. CLOUD VA HEALTH CARE SYSTEM"
+        "label": "St. Cloud VA Medical Center"
       }
     ],
     "IA": [
       {
         "value": "438GA",
-        "label": "SPIRIT LAKE CBOC"
+        "label": "Spirit Lake VA Clinic"
       },
       {
         "value": "636A8",
-        "label": "IOWA CITY HCS"
+        "label": "Iowa City VA Medical Center"
```

Comment:
Flagging this label since it goes from an entire health care system to a single facility.